### PR TITLE
[RW-4359][risk=no] Not sure if this is actually a good idea: Deprecate by renaming

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Comment by jmthibault79** [comment]
This may be a sledgehammer: I'm also trying (elsewhere) the route of thorough commenting with no functional code changes.

File: api/src/main/resources/workbench.yaml [hunk 1]
```diff
@@ -3629,10 +3629,11 @@ definitions:
         type: string
       areaOfResearch:
         type: string
-      institutionalAffiliations:
+      deprecatedInstitutionalAffiliations:
```
Comment (calbach) [inline api/src/main/resources/workbench.yaml]: If we do a process like this, we shouldn't change wire format (model names are fine, but not property names like this)

Comment (jaycarlton) [reply api/src/main/resources/workbench.yaml]: A couple of things can be marked deprecated on the swagger side as well https://swagger.io/docs/specification/describing-parameters/

I'm kinda thinking the path of least resistance is to come up with a different, shorter name for the new stuff,  and then just kill this one all at once. Maybe just Affiliation for the object, etc.

**Comment by jmthibault79** [comment]
I don't think this will be necessary.

**Pull request closed without merging**